src/plugins/packageManager.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
+const Helper = require("../helper");
+const colors = require("colors/safe");
+const log = require("../log.js");
+const fs = require("fs-extra");
+const path = require("path");
+const child = require("child_process");
+const packageJson = require("package-json");
+const packagesPath = Helper.getPackagesPath();
+const packagesParent = path.dirname(packagesPath);
+const packagesConfig = path.join(packagesParent, "package.json");
+const packagesNpmrc = path.join(packagesParent, ".npmrc");
```

Comment:
What problem is this solving?

I can see a few downsides of doing this, so I want to make sure I understand the decision completely before having an opinion :)

For example, I don't really like the idea that commands our code runs relies on arguments that are stored in userland (the configuration folder). Just the idea that if we want to change those, we have to migrate the saved file and keep the migration code for a few versions, or tell people to change it themselves is... not great (again, assuming I'm understanding things correctly, that might change after your answer 😁).

Reply:
Yeah, that's a fair problem. Thankfully, @xPaw found that you can pass in `--userconfig` with a path to a file and that is used as an npmrc file. So we can keep it in thelounge app folder, and keep it up to date that way. That'll be better.

The reason I'm doing this is so we aren't passing the individual arguments to commands that don't use it. I'll explain that later on because we need to discuss that in a larger way as to why we are doing that.